bower.json

```diff
@@ -5,8 +5,12 @@
     "ember-cli-shims": "0.1.1",
     "ember-cli-test-loader": "0.2.2",
     "ember-qunit-notifications": "0.1.0",
-    "bootstrap": "~3.3.5",
-    "ember-flag-icon-css": "*"
+    "bootstrap": "3.3.2",
+    "ember-flag-icon-css": "*",
```

Comment:
Hmmm...I'm not sure how this one got added in. I remember I added in the package.json one because npm couldn't find `ember-flag-icon-css`